api/fbs/v1/query.fbs

```diff
@@ -0,0 +1,121 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace v1;
```

Comment:
```suggestion
namespace banyandb.v1;
```
This should not include the version only.

Reply:
`namespace` determines the package name of generated go file, so `v1` is correct here.

Reply:
I agree with Hongtao. But in any case, we are able to override the namespace when compiling.

Reply:
Then we determine the scope of gRPC? We made a mistake in OAP, I wouldn't let that happen again.

Anything related to the RPC interface must have skywalking or banyandb included.

Reply:
Since I added `--go-namespace` in https://github.com/apache/skywalking-banyandb/blob/main/api/fbs/Makefile#L22, insert `banyandb` before `v1` won't change generated go files' package. 

Let's add `banyandb` to `namespace` to help generate java files with the proper package name.